fv3gfs/util/quantity.py

```diff
@@ -269,12 +269,29 @@ def __init__(
             extent: number of points along each axis within the computational domain
             gt4py_backend: backend to use for gt4py storages, if not given this will
                 be derived from a Storage if given as the data argument, otherwise the
-                storage attribute is disabled and will raise an exception
+                storage attribute is disabled and will raise an exception. will raise
```

Comment:
Minor point: should `exception. will` be `exception. Will`?

Reply:
Changed, I'm always unsure what to do when starting with lowercase and not using periods, but then running into a longer description that needs multiple sentences.